File: MusicList/app/src/main/java/com/example/musiclist/rest/ApiInterface.java
package com.example.musiclist.rest;

import com.example.musiclist.model.MusicResult;

import io.reactivex.Observable;
import retrofit2.http.GET;

public interface ApiInterface {

    @GET("/search?term=classick&amp;media=music&amp;entity=song&amp;limit=50")
    Observable<MusicResult> getMusicDetails();
}

File: KotlinMusic/app/src/main/java/com/example/kotlinmusic/rest/ApiInterface.kt
package com.example.kotlinmusic.rest

import com.example.kotlinmusic.model.MusicResult
import io.reactivex.Observable
import retrofit2.http.GET


interface ApiInterface {
    @GET("/search?term=classick&amp;media=music&amp;entity=song&amp;limit=50")
    fun getMusicDetails(): Observable<MusicResult>
}